src/arena/Sprint.js

```diff
@@ -373,117 +360,110 @@ function Sprint(props) {
                   />
                   <StatsBlock
                     statName={`${I18n.get("weekly")} CP`}
-                    score={`${props.redux.sprint[activeSprintId].teams[index].percentage}%`}
+                    score={`${props.redux.sprint[activeSprintId].teams[
+                      index
+                    ].percentage.toFixed(0)}%`}
                   />
                 </div>
-              </div>
+              </section>
 
-              <div
-                style={{
-                  display: "flex",
-                  flexWrap: "wrap",
-                }}
-              >
-                <section>
-                  <h2>{I18n.get("upcomingMission")}</h2>
-                  {upcomingMissions.length > 0 ? (
-                    <ul className="context-cards">
-                      {upcomingMissions.map((mission, i) => (
-                        <li
-                          className={flexCardClass}
-                          // eslint-disable-next-line react/no-array-index-key
-                          key={i}
-                          style={{
-                            display: "flex",
-                            flexDirection: "column",
-                            justifyContent: "space-between",
+              <section>
+                <h2>{I18n.get("upcomingMission")}</h2>
+                {upcomingMissions.length > 0 ? (
+                  <ul className="context-cards">
+                    {upcomingMissions.map((mission, i) => (
+                      <li
+                        className={flexCardClass}
+                        // eslint-disable-next-line react/no-array-index-key
+                        key={i}
+                        style={{
+                          display: "flex",
+                          flexDirection: "column",
+                          justifyContent: "space-between",
+                        }}
+                      >
+                        <div>
+                          {lengua !== "en" ? (
+                            <>
+                              <h3>{mission.esTitle}</h3>
+                              <p>{mission.esDescription}</p>
+                            </>
+                          ) : (
+                            <>
+                              <h3>{mission.title}</h3>
+                              <p>{mission.description}</p>
+                            </>
+                          )}
+                        </div>
+                        <Button
+                          onClick={() => {
+                            setShowProofModal(true);
+                            setActiveIndex(i);
+                            setActiveMission(mission);
                           }}
+                          disabled={status !== "active"}
                         >
-                          <div>
-                            {lengua !== "en" ? (
```

Comment:
A major refactor is in the works for the Sprint - a big part of why the sprint object is so big is because I am not just the task information 5 times, for each player, but also duplicating it in English and Spanish. This is a perfect example of technical debt that comes when you are trying to prototype something quickly, without thinking through it too well.